docs/reference/modules/transport.asciidoc

```diff
@@ -102,6 +102,20 @@ and ensuring that the keepalive interval is shorter than any timeout that might
 cause idle connections to be closed, or by setting `transport.ping_schedule` if
 keepalives cannot be configured.
 
+[float]
+==== Transport Compression
+
+By default Elasticsearch network-level compression is disabled. This default
+normally makes sense for local cluster communication as compression has a
+noticeable CPU cost and local clusters tend to be setup with fast network
+connections between nodes. If compression is necessary for remote cluster
+connections only, this can be configured on a per-cluster basis using the
```

Comment:
The "only" here seems to imply that setting transport.compress affects both internode and remote cluster communications. i.e. you set this cluster-level setting if you *only* want to compress remote cluster connections.  If that's not right, we should remove the "only".